packages/shared-ux/page/analytics_no_data/impl/src/analytics_no_data_page.component.tsx

```diff
@@ -79,6 +79,28 @@ const flavors: {
       },
     },
   }),
+  serverless_observability: ({ prependBasePath }) => ({
+    solution: i18n.translate('sharedUXPackages.noDataConfig.observability', {
+      defaultMessage: 'Observability',
+    }),
+    pageTitle: i18n.translate('sharedUXPackages.noDataConfig.observabilityPageTitle', {
+      defaultMessage: 'Welcome to Elastic Observability!',
+    }),
+    logo: 'logoObservability',
+    action: {
+      observability: {
+        title: i18n.translate('sharedUXPackages.noDataConfig.observabilityTitle', {
+          defaultMessage: 'Add data',
+        }),
+        description: i18n.translate('sharedUXPackages.noDataConfig.observabilityDescription', {
+          defaultMessage:
+            'Start collecting your log data to monitor the health of your applications',
```

Comment:
@sileschristian, @isaclfreire  could you please help with the text here? This is just what I came up with myself 

![Screenshot 2023-10-11 at 15 06 45](https://github.com/elastic/kibana/assets/7784120/5ceb5965-3209-4357-b471-e8efd23bbd59)

Reply:
Hi, assuming that the "Add data" button still goes to the Integrations page without any filter, I'd suggest something like this:
- Make the subtitle about Elastic Observability as they're visually paired together and also to not repeat what's in the empty state.
- Add a little more context to the empty state itself to hint on what's going to happen when they click (choose an integration to collect their O11y data).

<img width="687" alt="image" src="https://github.com/elastic/kibana/assets/10208282/789e1ceb-a008-42fe-a933-257bbb88426b">


Let me know if the above suggestion works for you. If that's too much, the proposed changes are also good enough for minimal changes. Or I can help adjust.

Reply:
@florent-leborgne looks good 🙇

Reply:
Thank you for the feedback 👍 
Updating!

![Screenshot 2023-10-17 at 12 12 53](https://github.com/elastic/kibana/assets/7784120/6b2bf2e4-0f52-4b80-8310-ce70d5132115)